docs/reference/snapshot-restore/repository-azure.asciidoc

```diff
@@ -140,6 +140,12 @@ stored in the keystore are marked as "secure"; the other settings belong in the
   set by the Azure client (known as 5 minutes). This setting can be defined
   globally, per account, or both.
 
+`endpoint`::
+  The Azure endpoint to connect to. It must include the protocol used to connect to Azure.
+
+`secondary_endpoint`::
```

Comment:
I've been trying to find the docs about the use case for the secondary endpoint, but couldn't find any specifics in Microsoft docs. Is it needed for redundancy?

Reply:
It's used for retries, see https://learn.microsoft.com/en-us/azure/storage/blobs/storage-retry-policy#use-geo-redundancy-to-improve-app-resiliency